e2e/npm_link_package-esm/MODULE.bazel

```diff
@@ -0,0 +1,6 @@
+###############################################################################
```

Comment:
Do we need this file?

Reply:
Opps, I guess that was me running tests with bzlmod and this e2e test isn't enabled with bzlmod? 🤷 

I'll drop the file before merging, can add it another day in its own PR if needed.